css/style.css

```diff
@@ -0,0 +1,18 @@
+#navbar {
+    position: fixed;
+    width: 300px;
+  }
+
+  #main-doc {
+    margin-left: 25%;
+  }
+
+@media only screen and (max-width: 800px) {
+    #main-doc{
+      margin-left: 1%;
+    }
+    #navbar {
+      position: relative;
+      width: auto;
+      }
+  }
```

Comment:
I think it's better for readibility to add two spaces of indentation instead of using the tab key, you can read more about HTML and CSS best practices on [this page](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).